lib/src/geofencing.dart

```diff
@@ -134,27 +118,22 @@ class GeofencingManager {
   /// `GeofenceRegion` provided only requests notifications for a
   /// `GeofenceEvent.dwell` trigger on iOS, `UnsupportedError` is thrown.
   static Future<void> registerGeofence(
-      GeofenceRegion region,
-      void Function(List<String> id, Location location, GeofenceEvent event)
-          callback) async {
-    if (Platform.isIOS &&
-        region.triggers.contains(GeofenceEvent.dwell) &&
-        (region.triggers.length == 1)) {
+      GeofenceRegion region, void Function(List<String> id, Location location, GeofenceEvent event) callback) async {
+    if (Platform.isIOS && region.triggers.contains(GeofenceEvent.dwell) && (region.triggers.length == 1)) {
       throw UnsupportedError("iOS does not support 'GeofenceEvent.dwell'");
     }
-    final List<dynamic> args = <dynamic>[
-      PluginUtilities.getCallbackHandle(callback).toRawHandle()
-    ];
+    final List<dynamic> args = <dynamic>[PluginUtilities.getCallbackHandle(callback).toRawHandle()];
     args.addAll(region._toArgs());
     await _channel.invokeMethod('GeofencingPlugin.registerGeofence', args);
   }
 
+  /// get all geofence identifiers
+  static Future<List<dynamic>> getRegisteredGeofenceIds() async => await _channel.invokeMethod('GeofencingPlugin.getRegisteredGeofenceIds');
```

Comment:
Please run `dartfmt` on your changes.

Reply:
My editor is set to run `dartfmt` on save. Disabled it and made `diff` as small as I could.

Reply:
`dartfmt` would wrap this onto another line (see the left of the diff for the expected format).